packages/compass-sidebar/src/components/favorite-button/favorite-button.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FavoriteIcon,
+  mergeProps,
+  useFocusRing
+} from '@mongodb-js/compass-components';
+
+import styles from './favorite-button.module.less';
+
+const FavoriteButton = ({
+  favoriteOptions,
+  toggleIsFavoriteModalVisible
+}) => {
+  const isFavorite = !!favoriteOptions;
+  const hex = favoriteOptions?.color;
+  const style = {
+    backgroundColor: hex || '#243642',
+    color: isFavorite ? '#ffffff' : '#88989a'
+  };
+
+  const focusRingProps = useFocusRing();
```

Comment:
This focus ring was not really built for this use-case and looks pretty weird because of that:

<img width="237" alt="image" src="https://user-images.githubusercontent.com/5036933/151334219-6121b61d-cff7-43d8-a7ca-cc6a08f90780.png">

You can override focus ring styles by adding some styles on top of that, but it needs to be done using `css`, because `less` styles will not take precedence over (due to the style insertion order). Something like this should do the trick:

```js
css({
    // -1 to compensate for the weird transparent border this button has
    top: -1,
    right: -1,
    bottom: -1,
    left: -1,
    borderRadius: 14,
})
```

Reply:
Removed the transparent border since it wasn't doing much but mess up these styles 👍 
<img width="144" alt="Screen Shot 2022-01-27 at 10 13 52 AM" src="https://user-images.githubusercontent.com/1791149/151387116-5749cfe8-1125-4cf6-b7eb-805f91daaa86.png">